.github/workflows/test.yml

```diff
@@ -158,13 +178,20 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
 
+      - name: install-hyperlane-cli
+        id: install-hyperlane-cli
+        uses: ./.github/actions/install-cli
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
       - name: Checkout registry
         uses: ./.github/actions/checkout-registry
 
       - name: CLI e2e tests (${{ matrix.test }})
         run: yarn --cwd typescript/cli test:e2e
         env:
           CLI_E2E_TEST: ${{ matrix.test }}
+          IS_CI: true
```

Comment:
instead of using `IS_CI`, just use `CI` - as github actions should automatically set this to true in CI https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/